mlflow/server/handlers.py

```diff
@@ -17,9 +17,9 @@
     DeleteExperiment, RestoreExperiment, RestoreRun, DeleteRun, UpdateExperiment
 from mlflow.store.artifact_repo import ArtifactRepository
 from mlflow.store.file_store import FileStore
+from mlflow.tracking.utils import _is_db_uri
```

Comment:
nit: just noticed that "db" is slightly confusing, as there are both database and databricks stores (where the latter is also sometimes shortened to `db_` in mlflow code). Maybe we could change it to `_is_database_uri` to avoid any sort of confusion along these lines.